psydac/api/feec.py

```diff
@@ -86,51 +86,88 @@ def __init__(self, mapping, *spaces):
     #--------------------------------------------------------------------------
     @property
     def dim(self):
+        """dimension of the physical and logical space"""
```

Comment:
I guess you mean: dimension of the physical and logical _domain_ ?

Reply:
and you can specify: (assuming both are of same dimension) since one could have dim(logical_Omega) < dim(physical_Omega)

Reply:
In other parts of Psydac we use the properties `ldim` for the logical domain and `pdim` for the physical domain. The FEEC part of Psydac always assumes that `pdim == ldim`.

Reply:
Fixed in 8e2067f